### PR TITLE
Use safe_urlopen instead of httplib to do replay

### PR DESCRIPTION
**Pull request by messense**
The requests package is much more nice to work with than httplib.

### CODE REVIEW

**Comment by mattrobenolt** [comment]
Hey @messense, thanks for the PR! :) Could you switch this to use `sentry.http` module instead? That module leverages `requests` and provides some other conveniences that we should be using here anyways.

**Comment by messense** [comment]
Sure, just switched it.

File: src/sentry/replays.py [hunk 1]
```diff
@@ -1,10 +1,6 @@
 from __future__ import absolute_import
-
-import socket
-
-from httplib import HTTPConnection, HTTPSConnection
-from urllib import urlencode
-from urlparse import urlparse
+from requests import RequestException
```
Comment (mattrobenolt) [inline src/sentry/replays.py]: Isn't this `from requests.exceptions import RequestException`?

That's how we use it elsewhere.

Comment (mattrobenolt) [inline src/sentry/replays.py]: Oh, I guess it gets imported into the root package too: https://github.com/kennethreitz/requests/blob/master/requests/__init__.py#L63-L66

Comment (messense) [inline src/sentry/replays.py]: Surely it is, but ``requests`` also expose ``RequestException`` to ``requests`` module level for convenience.

**Comment by mattrobenolt** [comment]
thx :cake: :sparkles:

**Pull request merged**